contracts/anchors/bridged/AnchorPoseidon2.sol

```diff
@@ -92,8 +94,9 @@ abstract contract AnchorPoseidon2 is MerkleTreePoseidon, ReentrancyGuard {
     uint32 insertedIndex = _insert(_commitment);
     commitments[_commitment] = true;
     _processDeposit();
+    depositNonce++;
+    emit Deposit(_commitment, insertedIndex, updateNonce, block.timestamp);
```

Comment:
```suggestion
    emit Deposit(_commitment, insertedIndex, depositNonce, block.timestamp);
```

Reply:
So the reason the nonce is not necessarily needed is that insertedIndex == depositNonce. We're duplicating this value afaik and can use the index interchangeably as a nonce value for the # of deposits that have taken place. I would consider removing this value and swapping the block.timestamp out for the newly updated merkle root too, since that data is incident in the update proposal. Thoughts?

Reply:
I agree with your point about the nonce being redundant when we have insertedIndex so I will delete those insertions. It would be good to emit the newest merkleroot so that relayers dont have to query it. In terms of removing block.timestamp, is it used to calculate anything (height) or not essential to the protocol? I am ambivalent about removing it if it is not used in any way, you would know more than I whether relayers would still appreciate it.